src/main/java/mods/railcraft/common/carts/EntityTunnelBore.java

```diff
@@ -857,6 +865,7 @@ protected float getLayerHardness(BlockPos targetPos, BlockRailBase.EnumRailDirec
             IBoreHead head = (IBoreHead) boreSlot.getItem();
             float dig = 2f - head.getDigModifier();
             hardness *= dig;
+            hardness /= (1 + EnchantmentHelper.getEnchantmentLevel(Enchantments.EFFICIENCY, boreSlot));
```

Comment:
I cannot match this algorithm with the numbers you posted.

Level 1 cuts the time in half and you get dismissing returns from there.